src/lib/coda_networking/coda_networking.ml

```diff
@@ -626,7 +626,12 @@ module Rpcs = struct
     | Consensus_rpc : ('q, 'r) Consensus.Hooks.Rpcs.rpc -> ('q, 'r) rpc
```

Comment:
the comment for ADDING A NEW RPC should probably mention updating `match_handler`